.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+If you found a bug in KLEE, please provide:
+1) A clear and concise description of what the bug is.
```

Comment:
That's not easy - I would go for something like:
1) What happened
2) What did you expect
...

Output of `klee --version`